v0/pipeline/stages/cd/service-hook.yaml

```diff
@@ -143,3 +143,11 @@ allOf:
     properties:
       store:
         $ref: ../../steps/cd/s3-url-store-config.yaml
+- if:
+    properties:
+      type:
+        const: HarnessCode
+  then:
+    properties:
+      spec:
+        $ref: harness-code-store.yaml
```

Comment:
do we need to integrate with service hook right now? we only support inline right now (functionally at least, not sure why the above schema supports all stores)

Reply:
yeah, i dont know why we did all in first place, do you want me to remove this ?

Reply:
no harm in keeping it I suppose. it will be needed in the future anyway if it has to be supported